src/dispatch/incident/messaging.py

```diff
@@ -338,6 +340,8 @@ def send_incident_update_notifications(incident: Incident, previous_incident: In
                 ticket_weblink=incident.ticket.weblink,
                 faq_weblink=incident.incident_faq.weblink,
                 incident_id=incident.id,
+                incident_priority_old=previous_incident.incident_priority.name,
+                incident_priority_new=incident.incident_priority.name,
```

Comment:
Without looking at the `notification_template`, looks like the following would also be missing:
```
incident_type_old=previous_incident.incident_type.name,
incident_type_new=incident.incident_type.name,
incident_status_old=previous_incident.status.value,
incident_status_new=incident.status,
```

Reply:
I'll take a look at it today.

Reply:
I'll take a look as well.  Only noticed that priority update emails had "** to **" instead of something like "*Low* to *Medium*"

Reply:
Just tested, will add these as well

Reply:
Awesome, thanks!

Reply:
Looking at `convo_plugin.send()` above, we can probably remove the following for this notification:
```
priority=incident.incident_priority.name,
document_weblink=incident.incident_document.weblink,
storage_weblink=incident.storage.weblink,
faq_weblink=incident.incident_faq.weblink,
```